encass-annotations/src/main/gateway/config/encass.yml

```diff
@@ -0,0 +1,32 @@
+encass_with_annotation:
```

Comment:
Create two encases:
1. hello-world
    it should have the following assertions
     - audit "hello world"
     - set response to "hello world"
2. find-internal-users
    it should have only one assertion
    - perform jdbc query

Reply:
Done